src/applications/personalization/profile-2/sass/profile-2.scss

```diff
@@ -82,3 +82,37 @@ nav.profile-side-nav {
     margin: unset;
   }
 }
+
+
+.headerNameWrapper {
+  width: 60%;
+  h2 {
+    text-transform: capitalize;
```

Comment:
let's remove this `text-transform` so that in the future when we get the name from `hero.userFullName` we aren't transforming it at all (there are names like `van der Molen` that we don't want to capitalize incorrectly)